### PR TITLE
Move 'This device' string to strings.xml

### PR DESCRIPTION
**Pull request by dmitriy-ilchenko**
I moved 'This device' string to strings.xml. Also the string "Device name" was also hardcoded.
#26

### CODE REVIEW

**Comment by csicar** [comment]
That looks awesome! Thank you for implementing that. I'll take a look at the code later

File: app/src/main/res/layout/fragment_deviceinfo_list.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
                             android:id="@+id/textView4"
                             android:layout_width="wrap_content"
                             android:layout_height="wrap_content"
-                            android:text="Device Name"
+                            android:text="@string/field_name_device_name"
```
Comment (csicar) [inline app/src/main/res/layout/fragment_deviceinfo_list.xml:49]: Thanks I totally missed that

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -9,8 +9,10 @@
     <string name="title_network_overview">Red {interface_name}</string>
     <string name="title_device_detail">Dispositivo {deviceIp}</string>
     <string name="title_card_device_info">Información del dispositivo</string>
+    <string name="field_name_device_name">Device Name</string>
```
Comment (csicar) [inline app/src/main/res/values-es/strings.xml:12]: If the translation is not known, just remove these fields from the translation.

That way, it's easier to find missing translations when somebody wants to resolve them

**Review by csicar — COMMENTED** [review]
Besides that, everything looks good :+1:

**Comment by dmitriy-ilchenko** [comment]
I removed unstranslated strings, everything should be OK.

**Pull request merged**